components/next_steps_view/steps.ts

```diff
@@ -135,11 +155,35 @@ export const showNextSteps = createSelector(
     },
 );
 
+// Only show tips if they have been skipped, or there are no unfinished steps
+export const showNextStepsTips = createSelector(
+    (state: GlobalState) => getCategory(state, Preferences.RECOMMENDED_NEXT_STEPS),
+    (state: GlobalState) => getLicense(state),
+    (state: GlobalState) => nextStepsNotFinished(state),
+    (stepPreferences, license, nextStepsNotFinished) => {
+        if (stepPreferences.some((pref) => (pref.name === RecommendedNextSteps.SKIP && pref.value === 'true'))) {
+            return true;
+        }
+
+        if (license.Cloud !== 'true') {
+            return false;
+        }
+
+        return !nextStepsNotFinished;
+    },
+);
+
+// Loop through all Steps. For each step, check that
 export const nextStepsNotFinished = createSelector(
     (state: GlobalState) => getCategory(state, Preferences.RECOMMENDED_NEXT_STEPS),
     (state: GlobalState) => getCurrentUser(state),
-    (stepPreferences, currentUser) => {
-        const checkPref = (step: StepType) => stepPreferences.some((pref) => (pref.name === step.id && pref.value === 'true') || !isStepForUser(step, currentUser.roles));
+    (state: GlobalState) => isFirstAdmin(state),
+    (stepPreferences, currentUser, firstAdmin) => {
+        let roles = currentUser.roles;
+        if (!firstAdmin) {
+            roles = 'system_user';
```

Comment:
It feels like using the roles isn't as reliable a way of determining system admin at this point.
It's not something we have to do now, but refactoring to work off of a single selector to check for `isFirstAdmin` would be ideal.

Reply:
isFirstAdmin is a single selector, the line you're referring to here basically changes which steps are shown based on that isFirstAdmin selector. I agree we could probably refactor this in the future to be a bit better - as is though, isFirstAdmin is a stop-gap, so I don't think worth fixing as part of this PR.